ui/components/calendar.js

```diff
@@ -0,0 +1,315 @@
+import moment from 'moment'
+import React, { useRef, useEffect, useState } from 'react'
+import { ChevronLeftIcon, ChevronRightIcon } from '@heroicons/react/outline'
+
+const monthName = [
```

Comment:
Is the logic in this file adapted from a different time library? Seems like there is some functionality here we don't need. If we adapted it, it would be nice to credit the original library in a comment at the top of the file and mention their license.